.github/workflows/im-build-test-deploy.yaml

```diff
@@ -324,3 +324,28 @@ jobs:
           KUBECONFIG: ${{ env.KUBECONFIG }}
           AWS_ACCESS_KEY_ID: ${{ secrets.AWS_ACCESS_KEY_ID }}
           AWS_SECRET_ACCESS_KEY: ${{ secrets.AWS_SECRET_ACCESS_KEY }}
+
+      - name: Add deployed environment URL as comment
+        uses: actions/github-script@v6
+        if: ${{ env.DEPLOY_ENVIRONMENT == 'true' }}
+        with:
+          script: |
+            let app_url
+
+            switch (context.repo.repo) {
+              case 'im-manager':
+                app_url = `https://${{ env.API_HOSTNAME }}`
```

Comment:
Not sure if it is a possibility to use the same `env` var accross repos 🤷🏻 We could then  comment `App deployed to ${env.APP_HOSTNAME}`

Reply:
It is a possibility, indeed! 🤦 

I think we can just change to `APP_HOSTNAME` (or just `HOSTNAME`) in both the [im-manager](https://github.com/search?q=repo%3Adhis2-sre%2Fim-manager%20API_HOSTNAME&type=code) and in the [im-web-client](https://github.com/search?q=repo%3Adhis2-sre%2Fim-web-client%20UI_HOSTNAME&type=code). 🤔 

@tonsV2 what do you think?

Reply:
The problem is that we need both in the im-tooling project.

Reply:
We could still have both in the respective repos/workflows but they could feed a common env that is picked up by this workflow.

Reply:
After further discussions it doesn't seem like there's an easy way to do this with the lack of env variables cross-referencing when setting variables in GHA and having to explicitly set the `_HOSTNAME` variables for each of the apps.